pkg/pilot/cassandra/v3/pilot.go

```diff
@@ -126,6 +165,10 @@ func (p *Pilot) ReadinessCheck() error {
 }
 
 func (p *Pilot) LivenessCheck() error {
+	if p.decommissionInProgress || true {
+		glog.Info("decommission in progress, reporting success for liveness")
+		return nil
+	}
```

Comment:
Yikes, sorry about this.
I remember you saying that jolokia nodetool requests fails if the node has been decommissioned.
I wonder if we need to rethink this check.